app/assets/stylesheets/hackney_property.scss

```diff
@@ -0,0 +1,6 @@
+.hackney-property-tenure {
+  background: $govuk-focus-colour;
+  padding: 4px 8px;
+  color: white !important;
+  width: 50%;
+}
```

Comment:
from trello:
> - Colour code the displayed field green for Tenant, orange for everything else (only implement this if it's not free text)

Reply:
yess, but this field is kinda free text. Btw I just asked vlad to add the tenure code to the API so we can do that.

Reply:
will put WIP tag on the PR